lxd/storage/drivers/driver_ceph_utils.go

```diff
@@ -1157,20 +1148,9 @@ func (d *ceph) sendVolume(conn io.ReadWriteCloser, volumeName string, volumePare
 	output, _ := io.ReadAll(stderr)
 
 	// Handle errors.
-	errs := []error{}
-	chStdoutPipeErr := <-chStdoutPipe
-
 	err = cmd.Wait()
 	if err != nil {
-		errs = append(errs, err)
-
-		if chStdoutPipeErr != nil {
-			errs = append(errs, chStdoutPipeErr)
-		}
-	}
-
-	if len(errs) > 0 {
-		return fmt.Errorf("ceph export-diff failed: %v (%s)", errs, string(output))
+		return fmt.Errorf("ceph export-diff failed: %v (%s)", err, string(output))
```

Comment:
Would it make sense here to use `%w` since it's now only a single error?

Reply:
Ah yeah we can do that now its been simplified to a single error.